server/app.js

```diff
@@ -212,8 +213,10 @@ module.exports = function createApp({ signInService }, logger, calendarService,
     })(req, res, next),
   )
 
-  app.use('/logout', (req, res) => {
+  app.use('/logout', async (req, res) => {
     if (req.user) {
+      await userAuthenticationService.updateSessionExpiryDateTime(req.user.username)
```

Comment:
not following this change - why do you need to modify track the session expiry time?  Isn't this just the expiry time of the cookie?

Reply:
Where is the expiry time of cookie set?